### PR TITLE
Update test.js

### PR DESCRIPTION
**Pull request by XenBG**
Added working methods and events, and removed everything else. Use it as a base code and upgrade it with the other methods and events.

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -162,13 +149,15 @@ try {
     if (sentry.length) logOnDetails.sha_sentryfile = sentry;
 }
 catch (beef){
-    util.log("Cannae load the sentry. " + beef);
+    util.log("Cannot load the sentry. " + beef);
```
Comment (jimmydorry) [inline test/test.js]: @RJacksonm1 Ye cannae tae mah eyron bruu!!

**Comment by jimmydorry** [comment]
Would have preferred it in dev.

File: test/test.js (hunk 1 above)
Comment (XenBG) [inline test/test.js]: @jimmydorry Vat laguang si dis?

Comment (RJacksonm1) [inline test/test.js]: Omg which monster did this.

Comment (jimmydorry) [inline test/test.js]: There is this great thing called `git blame`. It allows us to apportion blame for specific lines of code, like this one:

http://i.imgur.com/dXSEXCC.png

What a monster, indeed.

Comment (RJacksonm1) [inline test/test.js]: No that guy is fine.  Which monster REPLACED it is what I'm asking!

Comment (XenBG) [inline test/test.js]: Make a special pull request with returning of the "Cannae".

Don't let your dreams be dreams. Yesterday, you said tomorrow. So JUST DO IT. MAKE YOUR DREAMS COME TRUE.

```Kappa```

Comment (RJacksonm1) [inline test/test.js]: Nah it's OK.  This script will soon be replaced by our robot overlords #147.

`Cannae` will return with basic examples though!

Comment (XenBG) [inline test/test.js]: So I'm not the monster.

Comment (RJacksonm1) [inline test/test.js]: No you're still a monster.

File: test/test.js [hunk 2]
```diff
@@ -8,149 +8,136 @@ var steam = require("steam"),
...
-        util.log("sentryfile saved");
+    fs.writeFileSync('sentry', sentry.bytes)
+    util.log("sentryfile saved");
+
     callback({ sha_file: crypto.createHash('sha1').update(sentry.bytes).digest() });
 });
 
-
-// Login, only passing authCode if it exists
 var logOnDetails = {
```
Comment (howardchung) [inline test/test.js]: should use process.env.  Either use .env and dotenv to load or have user pass them in on the command line.

File: test/test.js (hunk 1 above)
Comment (howardchung) [inline test/test.js]: might want to rename this to example.js so we dinnae have merge conflicts with #147

Comment (jimmydorry) [inline test/test.js]: It's already fubarred sync wise. I'll just make a note to fix it up when I have to do the re-base and merge.

File: test/test.js (hunk 2 above)
Comment (jimmydorry) [inline test/test.js]: It's just an example. Unless it doesn't work (in which case you should PR). 

I would hope developers would use this as a reference only, and implement what ever best practice is or what ever takes their fancy.

Comment (howardchung) [inline test/test.js]: Sure, but does that mean you need to keep a reference to global.config around (or else the example won't work as-is)

Comment (jimmydorry) [inline test/test.js]: I can appreciate what you mean and understand where you are coming from. However, if they copy paste this without referencing the config, then they should get a pretty easy to understand error. If they can't figure out why it's failing, I doubt they will be able to do much with the library. :)

I'll gladly merge any improvements, but I'm happy enough for it to stay as is. I certainly prefer a config file over commandline, but that is just my personal preference as I don't plan to run lots of instances.

**Pull request merged**